module.go

```diff
@@ -391,3 +391,33 @@ func (m *Module) Iterator() *BPFObjectIterator {
 		prevMap:  nil,
 	}
 }
+
+func (m *Module) AttachProgs() error {
```

Comment:
A comment for this method is welcome, explaining the emulation for bpf_object__attach_skeleton

Reply:
some comments were added.